packages/block-editor/src/components/tool-selector/README.md

```diff
@@ -0,0 +1,37 @@
+# Tool Selector
+
+Tools provide different interactions for selecting, navigating, and editing blocks.
```

Comment:
Hi @Sidsector9, at first sight reading this phrase does not give a clear idea of what the component does or what its UI renders concretely. Would you be able to expand this introduction text a bit more ? I really appreciate any help you can provide.